src/plotman/manager.py

```diff
@@ -101,7 +102,7 @@ def maybe_start_new_plot(dir_cfg, sched_cfg, plotting_cfg):
                 dstdir = max(dir2ph, key=dir2ph.get)
 
             logfile = os.path.join(
-                dir_cfg.log, datetime.now().strftime('%Y-%m-%d-%H:%M:%S.log')
+                dir_cfg.log, pendulum.now().strftime('%Y-%m-%dT%H_%M_%S.%f%z.log')
```

Comment:
I was thinking we would literally use an ISO format and replace the colons.  Doesn't super matter, but since I checked to see what it would look like...  Thoughts?

```suggestion
                dir_cfg.log, pendulum.now().isoformat(timespec='microseconds').replace(':', '_') + '.log'
```

Reply:
Ah yeah, way nicer 👍

Reply:
Any documentation on preferred way of handling revisions like this? I can use the github "Commit Suggestion" action here and then squash and reopen? Or force-push? At my job we tend to avoid force-pushing if there's been any line-comments

Reply:
I default to not modifying history etc.  I just keep committing and going and so on.  And yeah, I use the suggestion so people can just click the button to commit if they like it.  Shouldn't be any need to squash etc.